app/controllers/searches_controller.rb

```diff
@@ -3,10 +3,7 @@ class SearchesController < ApplicationController
 
   def show
     @q = params[:q]
-    @journeys = Journey.published.search(@q).limit(5)
-    @steps = Step.published.search(@q).limit(5)
-    @pages = Page.faq.search(@q).limit(5)
-
-    @count = @journeys.length + @steps.length + @pages.length
+    analyzed_q = Transliterator.transliterate(@q)
+    @searches = PgSearch.multisearch(analyzed_q).page(params[:page]).per(2)
```

Comment:
To `per(2)` asi ostalo z nejakeho testu nie?

Reply:
jj fixnute na 10